tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "allowJs": true,
+    "downlevelIteration": true,
```

Comment:
This allows us to iterate over `Set` and other iterable objects like we can do with arrays.

Reply:
Is it possible to add a comment for this as well?

Reply:
Unfortunately not, it seems that JSON does not support comments in their files (https://stackoverflow.com/questions/244777/can-comments-be-used-in-json).